server/models/service-process.js

```diff
@@ -0,0 +1,132 @@
+var async = require('async');
+var debug = require('debug')('strong-mesh-models:server:service-process');
+
+module.exports = function(ServiceProcess) {
+  function recordFork(instanceId, pInfo, callback) {
+    debug('Process forked: worker id %d pid %d ppid %d',
+      pInfo.id, pInfo.pid, pInfo.ppid);
+
+    var proc = new ServiceProcess({
+      pid: pInfo.pid,
+      parentPid: pInfo.ppid,
+      workerId: pInfo.id,
+      serviceInstanceId: instanceId,
+      startTime: new Date()
```

Comment:
If this API can be used remotely, doesn't that make this `startTime` actually refer to the serialization time (local to the server) rather than the actual start time (local to the actual host running the process)?